### ISSUE TITLE
How to correctly use common.webExtension needs documenting

### ISSUE THREAD

**Issue by raintonr** [ioBroker/ioBroker.web#57]
I notice from looking at other adapters (ie. ioBroker.simple-api) that one can have code run inside the web server this ioBroker.web adapter starts.

However, this doesn't seem to be documented anywhere.

If developers are to use the features provided by this adapter correctly correct documentation should be supplied.

**Comment by Apollon77** [comment]
I completely agree. simpleapi and also parser are two examples ... If you learn it we are also happy to get a PR for such a documentation - or even more detailed information and/or a skeleton to enhance the adapter-creator :-)

**Comment by GermanBluefox** [comment]
https://github.com/ioBroker/ioBroker.web/blob/master/WEB-EXTENSIONS-HOWTO.md
